modyn/trainer_server/internal/dataset/key_sources/__init__.py

```diff
@@ -3,14 +3,11 @@
 """
 
 import os
-from enum import Enum
 
 from .abstract_key_source import AbstractKeySource  # noqa: F401
 from .local_key_source import LocalKeySource  # noqa: F401
 from .selector_key_source import SelectorKeySource  # noqa: F401
 
-KeySourceNames = Enum("KeySource", ["LOCAL_DOWNSAMPLER", "SELECTOR"])
-
```

Comment:
not used anywhere, and complained by mypy for different names (KeySourceNames vs KeySource)